src/test/java/com/purbon/kafka/topology/AccessControlManagerTest.java

```diff
@@ -76,24 +73,15 @@ public void setup() throws IOException {
 
   @Test
   public void newConsumerACLsCreation() {
-
-    List<Consumer> consumers = new ArrayList<>();
-    consumers.add(new Consumer("User:app1"));
-    Project project = new ProjectImpl();
-    project.setConsumers(consumers);
-
     Topic topicA = new TopicImpl("topicA");
-    project.addTopic(topicA);
-
-    Topology topology = new TopologyImpl();
-    topology.addProject(project);
-
-    List<Consumer> users = asList(new Consumer("User:app1"));
+    TestTopologyBuilder builder =
```

Comment:
If I understand the logic and idea correctly, this object is a wrapper for the Topology, but the name can be a bit confusing (at least it was to me).

As it looks, this object does a few things:

* Output a plain topology object.
* Have accessor helpers for Consumers and other related properties.

would it be better to name it differently? I like what it does, but the name and the fact that the variable is named builder makes the flow strange to me.

Reply:
It is a builder of (simple) topologies for testing. Maybe a bit more advanced than most of the builders you see around, but still definitively a builder.

In general there is nothing wrong with being able to both set and get values on a builder. So I still think it is a builder even though you can get back the values you have added.

Reply:
Du you have any suggestion for other naming?

Reply:
Only this discussion pending now. Name can also be changed later? Unless you have some other suggestion? 

Would be nice to get this PR through quite fast as I will also need the builder when implementing #117 .

Reply:
@purbon ☝️

Reply:
Technically you are right, however, the name is still confusing. I will add an issue as a follow up to improve.

Reply:
@purbon , yes I can understand that is a bit confusing, since there are a few *TopologyBuilder around. However as you understand, wasn't able to come up with something better/different :-) Let's see if we can improve it later.